src/containers/shared/components/TxDetails.tsx

```diff
@@ -54,28 +56,19 @@ interface Instructions {
 
 interface Props {
   instructions: Instructions;
-  type: string | null;
-  language: string;
-  t: (s: string) => string;
+  type: string;
 }
 
 const TxDetails = (props: Props) => {
-  function renderAmount(d: any): ReactElement {
-    const { language } = props;
-    const options = { ...CURRENCY_OPTIONS, currency: d.currency };
-    const amount = localizeNumber(d.amount, language, options);
-    const { issuer, currency } = d;
+  const language = useLanguage();
+  const { t } = useTranslation();
 
-    return (
-      <span className="amount">
-        {amount}
-        <Currency issuer={issuer} currency={currency} link={false} />
-      </span>
-    );
+  function renderAmount(d: any): ReactElement {
+    return <Amount value={d} />;
   }
```

Comment:
This seems like a useless helper function - why not just use `Amount` normally?

Reply:
I didnt want to change any of the other transactions just yet.  I wanted to make as few changes as possible.  Once those get updated I will delete this.